examples/complete_workflow/AuthoritationCodeCompleteWorkFlow.java

```diff
@@ -0,0 +1,157 @@
+import com.wrapper.spotify.model_objects.specification.PagingCursorbased;
+import com.wrapper.spotify.model_objects.specification.PlayHistory;
+import com.wrapper.spotify.model_objects.specification.SavedAlbum;
+import com.wrapper.spotify.model_objects.specification.Track;
+import com.wrapper.spotify.model_objects.specification.User;
+import com.wrapper.spotify.requests.authorization.authorization_code.AuthorizationCodeRefreshRequest;
+import com.wrapper.spotify.requests.authorization.authorization_code.AuthorizationCodeRequest;
+import com.wrapper.spotify.requests.authorization.authorization_code.AuthorizationCodeUriRequest;
+import com.wrapper.spotify.requests.data.albums.GetAlbumRequest;
+import com.wrapper.spotify.requests.data.artists.GetArtistsTopTracksRequest;
+import com.wrapper.spotify.requests.data.artists.GetArtistsTopTracksRequest.Builder;
+import com.wrapper.spotify.requests.data.library.GetCurrentUsersSavedAlbumsRequest;
+import com.wrapper.spotify.requests.data.player.GetCurrentUsersRecentlyPlayedTracksRequest;
+import com.wrapper.spotify.requests.data.player.StartResumeUsersPlaybackRequest;
+import com.wrapper.spotify.requests.data.search.simplified.SearchAlbumsRequest;
+import com.wrapper.spotify.requests.data.search.simplified.SearchArtistsRequest;
+import com.wrapper.spotify.requests.data.search.simplified.SearchTracksRequest;
+import com.wrapper.spotify.requests.data.users_profile.GetCurrentUsersProfileRequest;
+
+public class AuthoritationCodeCompleteWorkFlow {
+	/*
+	* Before testing the API you must register your application on the Spotify Developers DashBoard. 
+	* After registering, copy your cient_id and secret_id and paste them into the following constants. 
+	* You will also need to have a redirect address, to which the user will be redirected after authorisation 
+	* to use the app, which must be authorised in the Spotify Developer DashBoard in the configuration area. 
+	* In the URI after the redirection there will be a code that will be used to exchange it for access and 
+	* refresh tokens and get data from Spotify.
+	*/
+	
+	private static final String clientId = "zyuxhfo1c51b5hxjk09x2uhv5n0svgd6g";
+	  private static final String clientSecret = "zudknyqbh3wunbhcvg9uyvo7uwzeu6nne";
+	  private static final URI redirectUri = SpotifyHttpManager.makeUri("https://example.com/spotify-redirect");
+
+	  
+	  /*
+	   * Add the properties to our spotifyApi object
+	   */
+	  private static final SpotifyApi spotifyApi = new SpotifyApi.Builder()
+	    .setClientId(clientId)
+	    .setClientSecret(clientSecret)
+	    .setRedirectUri(redirectUri)
+	    .build();
+	  
+	  /**
+	   * <p>After pasting the client_id and secret_id we can execute the fetching of the code that will later be exchanged for an access token and an refresh token.
+	   * The scope field indicates what we will be able to do with the user's information. This will be shown to them before they authorise our app.</p>
```

Comment:
```suggestion
	   * The scope field indicates what can be done with the user's information. This will be shown to them before they authorize our app.</p>
```